### PR TITLE
Fixed issue #2 on PR #4 - Network and IO calls being made on MAIN thread

### PR DESCRIPTION
**Pull request by rajitdeb**
Fixed issue #2 on PR #4 - Network and IO calls being made on MAIN thread instead of IO thread

### CODE REVIEW

**Comment by rajitdeb** [comment]
Please review the changes and merge @PratikFagadiya

File: app/src/main/AndroidManifest.xml
Comment (PratikFagadiya) [inline app/src/main/AndroidManifest.xml:1]: It's always good practice to provide package name in manifest file

File: app/src/main/java/com/newsapp/withmvvm/api/NewsAPI.kt
Comment (PratikFagadiya) [inline app/src/main/java/com/newsapp/withmvvm/api/NewsAPI.kt:1]: Aha Interesting but what if i have multiple version with Same BASE URL ?? But anyways currently it looks fine to me

**Comment by PratikFagadiya** [comment]
Thanks @rajitdeb for showing interest and review code.

File: app/src/main/AndroidManifest.xml
Comment (rajitdeb) [reply app/src/main/AndroidManifest.xml:1]: > It's always good practice to provide package name in manifest file

Actually, it is the new way of declaring package name suggested by Gradle plugin itself

File: app/src/main/java/com/newsapp/withmvvm/api/NewsAPI.kt
Comment (rajitdeb) [reply app/src/main/java/com/newsapp/withmvvm/api/NewsAPI.kt:1]: Then we can just create a variable version in the Constants.kt file and based on the logic we can concatenate with the BASE URL

**Comment by rajitdeb** [comment]
> Thanks @rajitdeb for showing interest and review code.

No problem man. Btw you did a pretty good job as well

**Comment by PratikFagadiya** [comment]
Thank you @rajitdeb

**Pull request merged**